subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -389,11 +394,6 @@ static int init_reset(void)
 	return 0;
 }
 
-static inline struct ll_adv_sync_set *sync_acquire(void)
-{
-	return mem_acquire(&adv_sync_free);
-}
-
```

Comment:
It seems inconsistent to drop `sync_acquire` but keep `sync_release`. Please either keep both (which I would prefer) or drop both.

Reply:
I will revert.